### ISSUE TITLE
Sheet: moving columns messes up sort/filter display

### ISSUE THREAD

**Issue by jgeekmz** [primefaces-extensions/primefaces-extensions#911]
### Describe the bug

Sheet with several columns with sort and filter options. When you add movable option for rows and columns. On one column you do not have sort and filter. So when switch the place on this one with other column with filter and sort the header is moved, but the sort and filter options stayed on the same place as before. I guess this behaviour is bug.

Stack Overflow: https://stackoverflow.com/questions/73248207/primefaces-sheet-component-movable-columns

### Reproducer

```xml
<pe:sheet id="sheet" widgetVar="sheetWidget" value="#{vertragspreiseForm.lazyModel.getPreiseList()}" var="row"
        rowKey="#{row.pflegeartId}"
        height="400"
        resizableRows="true"
        resizableCols="true"
        fixedCols="2"
        showColumnHeaders="true"
        movableCols="true" movableRows="true">
```

### Expected behavior

When you move a column i expect to move with sort and filter optons as well.

### PrimeFaces Extensions version

7.0.3

### JSF version

2.0

### Browser(s)

Chrome

**Comment by jepsar** [comment]
Please add a reproducer

**Comment by melloware** [comment]
Updated with Stack overflow info.  My guess is these features were just never tested together as movable columns was added way after sorting/filtering.

**Comment by VsevolodGolovanov** [comment]
In a `cellSelect` listener `sheet.getSelectedColumn()` also reports wrong column index.

pe:sheet widget should do `hot.toPhysicalColumn(col)`, because moving columns changes their visual indexes (which is what most of HOT's API uses).

**Comment by melloware** [comment]
PR?

**Comment by jxmai** [comment]
@melloware I don't think we can fix this one without alternating Handsontable source code since there's limitation on Handsontable 6.2.2, even if physical dragging issue is applied as suggested by @VsevolodGolovanov  (That's already risky changes BTW), the drag preview is rendered by Handsontable ManualColumnMove backlight UI (https://raw.githubusercontent.com/primefaces-extensions/primefaces-extensions/refs/heads/master/core/src/main/resources/META-INF/resources/primefaces-extensions/sheet/0-handsontable.full.js) Upstream positions that overlay using row-header-aware offsets and event.layerX, so the preview can appear visually offset when row headers are enabled

In simple term, PrimeFaces Extensions keeps the actual column order and Sheet sort/filter bindings correct, but does not override Handsontable's private drag-preview UI.

**Comment by melloware** [comment]
Yep I mean we are stuck on Ht 6.2 since it's the last open source version so maybe hacking their source at this point isn't a bad idea?

I mean we haven't been able to upgrade in like 6 years because they changed their license!

**Comment by jxmai** [comment]
Let's update the source if that's the case. I included the source change with comments so it will be easy to identify.
